### ISSUE TITLE
Update the AssetImage component in the UI

### ISSUE THREAD

**Issue by LautaroPetaccio** [decentraland/marketplace#2234]
This ticket is part of the Simplify the AssetImage component initiative which has as its objective the simplification of the AssetImage component, making it exportable to other projects.

To complete this task:
- Update the AssetImage component in the UI to match any new changes that the AssetImage component has in the Marketplace UI.

**Comment by Jagadeeshftw** [comment]
I want to work on this issue, can you please assign this to me?

**Comment by LautaroPetaccio** [comment]
Hi @Jagadeeshftw , before starting this task, there's another task that needs to be completed, https://github.com/decentraland/ui/issues/533.

**Comment by Jagadeeshftw** [comment]
Hi @LautaroPetaccio, Since the UI and marketplace repositories are on different Node.js versions, I’m encountering dependency conflicts when trying to move assertImage from the marketplace repository to the UI. Do you have any suggestions?

**Comment by LautaroPetaccio** [comment]
Hi @Jagadeeshftw , what's causing this issue between the two versions?
Moving the `AssetImage` component from one repository to the other implies more than just copy pasting the code. We plan to migrate the UI logic into the UI repository, leaving the data consumption layer in the Marketplace repository, along some other refactors, like removing the [WearablePreview](https://github.com/decentraland/marketplace/issues/2233) from it. This issue kind of requires some context and the knowledge of the repositories. I would recommend you to start with issues under the [good first issue](https://github.com/decentraland/marketplace/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) category.
